source/includes/_node.md

```diff
@@ -2,106 +2,216 @@
 
 ## JSON-RPC Requests 
 
-```shell
-
-```
-
-```javascript
-
-```
+Route for JSON-RPC requests, most of which mimic the bitcoind RPC calls completely.
 
-> The above command returns JSON structured like this:
+### HTTP Request
+`POST /`
```

Comment:
Yeah, good call. What if we put one example here just so they can see how this endpoint works and then I can make a separate RPC section and we can do that as a separate task. Would be good to document that all ourselves rather than force people to go elsewhere to find a reference of bitcoin RPC calls.